docs/data-sources/table.md

```diff
@@ -0,0 +1,58 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_table Data Source
+
+-> **Note** This data source could be only used with workspace-level provider!
+
+-> **Note** If you have a fully automated setup with workspaces created by [databricks_mws_workspaces](../resources/mws_workspaces.md) or [azurerm_databricks_workspace](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/databricks_workspace), please make sure to add [depends_on attribute](../guides/troubleshooting.md#data-resources-and-authentication-is-not-configured-errors) in order to prevent _default auth: cannot configure default credentials_ errors.
+
+Retrieves details of a specific table in Unity Catalog, that were created by Terraform or manually. Use [databricks_tables](tables.md) to retrieve multiple tables in Unity Catalog
+
+## Example Usage
+
+Read  on a specific table `main.certified.fct_transactions`:
+
+```hcl
+data "databricks_table" "fct_transactions" {
+  name = "main.certified.fct_transactions"
+}
+
+resource "databricks_grants" "things" {
+  for_each = data.databricks_tables.things.ids
+
+  table = each.value
+
+  grant {
+    principal  = "sensitive"
+    privileges = ["SELECT", "MODIFY"]
+  }
+}
```

Comment:
This doesn't match the table ID, and it's not a list. We need to get something else here as an example.